FirebaseTestProject/ViewModels/SignInViewModel.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  SignInViewModel.swift
+//  SignInViewModel
+//
+//  Created by Tim Miller on 8/12/21.
+//
+
+
+import FirebaseAuth
+
+class SignInViewModel: ObservableObject {
+    
+    let auth = Auth.auth()
+    let userDefault = UserDefaults.standard
+    
+    @Published var signedIn = UserDefaults.standard.bool(forKey: "userSignedIn")
```

Comment:
`signedIn` -> `isSignedIn`

Reply:
Also can we use the user defaults property wrapper here?

Reply:
Changed to @AppStorage